deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md

```diff
@@ -85,3 +85,16 @@ By performing these steps, you essentially map the Packaging box from HotWax Com
 {% embed url="https://youtu.be/9XefekAuxr8" %}
 Video: Remove Shipment Box
 {% endembed %}
+
+### Add Career Facility
+
+In HotWax Commerce, the ability to add shipping carriers to facilities is essential for order fulfillment. Once a shipment gateway is set, adding carriers to facilities enables rate shopping and shipping label generation specifically for those carriers associated with the selected facility. This feature enhances workflow efficiency by allowing the HotWax Commerce Store Fulfillment App to intelligently select the most cost-effective shipping carrier for each order. To add careers to a facility follow these steps:
+
+1. Navigate to the `Career Facility` section.
```

Comment:
I don't think this is correct @Amir-HC

Reply:
<img width="1361" alt="Screenshot 2024-02-06 at 6 56 50 PM" src="https://github.com/hotwax/oms-documentation/assets/135032256/74397ffb-18d0-4809-a2f0-e06a27c35854">

I checked this on demo instance

Reply:
Check spelling of carrier